src/api/center.js

```diff
@@ -1,11 +1,11 @@
 const toArray = require('../core/utils/toArray')
 
-/** NOTE: this is not functional YET !!
- * centers the given object(s) on the given axis
+/**
+ * centers the given object(s) on the given axes
  * @param {Object|Array} object(s) the shapes to center
  * @param {Object} options
  */
-const center = (objects, options) => {
+const centerV2 = function (objects, options) {
```

Comment:
why rename it ?

Reply:
I didn’t like the fact that center() was calling center().

Reply:
centerV2 = function (options, objects) is correct

Reply:
but centerV2 is just awkward :)
one center is a method attached to prototype, the other one a simple function, I see no issues with it to be honest :)

Reply:
True. CSG.center() calls centerV2()

So does API center(). See ops-transformations.js